modules/dataquery/js/react.filterBuilder.js

```diff
@@ -64,9 +64,13 @@ FilterRule = React.createClass({
 				"startsWith": "startsWith",
 				"contains": "contains"
 				// }
-			}
+			},
+			value: ""
 		};
 	},
+	componentWillMount: function () {
+		this.valueSet = loris.debounce(this.valueSet, 1000);
```

Comment:
Its a JavaScript debounce function. It makes it so that when adding typing in a text box the onChange function isn't called every time the user inputs a new char